src/makeTxn.ts

```diff
@@ -408,10 +414,38 @@ export function makeApplicationCallTxnFromObject({
   lease,
   rekeyTo,
   suggestedParams,
-}: ApplicationCallTransactionParams & CommonTransactionParams): Transaction {
+}: ApplicationCallTransactionParams &
```

Comment:
Noting this is a bit different than the other SDKs; if the `convertToAccess` equivalent is false, then one cannot pass holdings/locals at all - there isn't a separate `access` param.

Reply:
please clarify, I probably did not get it - this params list has `access`:
```
  foreignApps,
  foreignAssets,
  boxes,
  convertToAccess,
  holdings,
  locals,
  access,
```